todo-app/src/TodoApp.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import TodoList from './TodoList';
+import todoItems from './todo.json';
+
+export default class TodoApp extends Component {
+
+  state = {
+    commentText: '',
```

Comment:
be careful to name your variables properly! This is a todo app, not the blog app.

Reply:
Changed 👍  @joostlubach 
maybe it was because  watching the video a lot :))